uber_analytics/api.py

```diff
@@ -0,0 +1,24 @@
+from uber.common import *
+
+from .site_sections.graphs import RegistrationDataOneYear
+
+class AnalyticsApi:
+    def badge_sale_history(self):
+        with Session() as session:
+            reg_data = RegistrationDataOneYear()
+            reg_data.query_current_year(session)
+
+            return {
+                'current_registrations': reg_data.dump_data(),
+            }
+
+    def badge_stats(self):
+        return {
+            'badges_sold': c.BADGES_SOLD,
+            'remaining_badges': c.REMAINING_BADGES,
+            'badges_price': c.BADGE_PRICE,
```

Comment:
I think we only ever used ```badges_price``` on the magstock site.  if this is a copy of what's in registration.py then you can name stuff here whatever you want.